capabilities/documentation-quality.md

```diff
@@ -1,51 +1,65 @@
 
-Documentation quality refers to the accuracy, completeness, and accessibility of internal documentation, enabling teams to effectively collaborate, make informed decisions, and deliver high-quality software quickly and reliably. By investing in documentation quality, developers can reduce errors, improve productivity, and enhance overall performance.
+_Documentation_ refers to written descriptions of internal decisions, processes, or policies. A team of software developers regularly creates and maintains documentation on a range of items, including technical design details, simple how-to cheat sheets, product requirements, pitfalls to be aware of in the tech stack, and testing plans. Whenever changes are made in code, it must be reflected in documentation. 
 
-## Nuance
+_Documentation quality_ refers to the accuracy, clarity, completeness, and accessibility of internal documentation. Excellent documentation enables teams to effectively collaborate, make informed decisions, and deliver high-quality software quickly and reliably. 
 
-### Importance of Clear and Findable Documentation
+When documentation is outdated or out of sync with the codebase, incomplete, inaccurate, or hard to find, teams suffer. The lack of guidance impedes productivity, frustrates developers, and greatly increases the chances of errors. So, to produce internal documents with care and quality, it's essential to understand what makes for good, effective documentation. 
 
-Clear and findable documentation is essential for effective knowledge sharing and collaboration within teams. Without it, teams may struggle to understand existing systems, leading to a decrease in development speed and reliability.
+## Nuances
+This section outlines common pitfalls, challenges, or limitations teams commonly encounter when applying this capability. The goal here is not to discourage you. Rather, the goal is to arm you with the appropriate context so that you can make an informed decision about when and how to implement the capability with your teams.
 
-### Reliability and Maintenance of Documentation
 
-Documentation must be actively maintained and updated to ensure it remains relevant and accurate. This requires a deliberate effort to review and refine documentation regularly. Sometimes misleading documentation can be worse than missing documentation.
+### Documentation Must Be Clear and Accessible
+
+Clear and accessible documentation is essential for effective knowledge sharing and collaboration within teams. Without it, teams may struggle to understand existing systems, leading to a decrease in development speed and reliability. Sometimes misleading documentation can be worse than missing documentation.
+
+### Documentation Must Be Reliable and Maintained
+
+Documentation must be actively maintained and updated to ensure it remains relevant, reliable, and accurate. This means that developers should make a deliberate effort to regularly review and refine documentation. 
```

Comment:
I'd like to flesh this out by defining (even broadly) what we mean by "regularly." e.g., "For some teams, this could mean a documentation audit every 4 weeks. For other teams, this could mean real-time updates to code changes."